puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -0,0 +1,61 @@
+def packages_to_build
```

Comment:
Should this be named ```rpm_packaging``` or ```packaging_rpm```?

Reply:
why? that will contain the list of packages to build later.

Reply:
I was looking at the filename but failed to write ```file``` in that sentence.

Reply:
fair enough.